test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -913,5 +913,60 @@ private void SetUserSecrets(string workingDir, Dictionary<string, string> userSe
                 _output.WriteLine(procOutput);
             }
         }
+
+        [Theory]
+        [InlineData("dotnet")]
+        [InlineData("node")]
+        [InlineData("javascript")]
+        public async Task start_on_missing_local_settings_json(string language)
+        {
+            await CliTester.Run(new RunConfiguration[]
+            {
+                new RunConfiguration
+                {
+                    Commands = new[]
+                    {
+                        $"init . --worker-runtime {language}",
```

Comment:
Can you write a test for when --worker-runtime is not provided and is None

Reply:
@liliankasem start to host function app when  local.settings.json is missing ,it will show 
![image](https://github.com/user-attachments/assets/11f38fce-7a92-4499-91fa-919739286d21)
then we can select the worker runtime

Reply:
@liliankasem could you please review overall changes after review. also added new test cases